chia/simulator/full_node_simulator.py

```diff
@@ -144,3 +193,250 @@ async def reorg_from_index_to_new_index(self, request: ReorgProtocol):
 
         for block in more_blocks:
             await self.full_node.respond_block(RespondBlock(block))
+
+    async def process_blocks(
+        self, count: int, farm_to: bytes32 = bytes32([0] * 32), guarantee_transaction_blocks: bool = False
+    ) -> int:
+        """Process the requested number of blocks including farming to the passed puzzle
+        hash. Note that the rewards for the last block will not have been processed.
+        Consider `.farm_blocks()` or `.farm_rewards()` if the goal is to receive XCH at
+        an address.
+
+        Arguments:
+            count: The number of blocks to process.
+            farm_to: The puzzle hash to farm the block rewards to.
+
+        Returns:
+            The total number of reward mojos for the processed blocks.
+        """
+        rewards = 0
+        height = uint32(0)
+
+        if count == 0:
+            return rewards
+
+        for _ in range(count):
+            block: FullBlock
+            if guarantee_transaction_blocks:
+                block = await self.farm_new_transaction_block(FarmNewBlockProtocol(farm_to))
+            else:
+                block = await self.farm_new_block(FarmNewBlockProtocol(farm_to))
+            height = uint32(block.height)
+            rewards += calculate_pool_reward(height) + calculate_base_farmer_reward(height)
+
+        peak_height = self.full_node.blockchain.get_peak_height()
+
+        if peak_height is None:
+            raise RuntimeError("Peak height still None after processing at least one block")
+
+        if peak_height < height:
```

Comment:
this is also impossible

Reply:
Disappeared in https://github.com/Chia-Network/chia-blockchain/commit/93c525cb999fa89d6d06d4fea78c9e81345912ca.